patches/server/0141-Basic-PlayerProfile-API.patch

```diff
@@ -103,16 +103,17 @@ index 0000000000000000000000000000000000000000..e513ce5bcb5070b8a57737228904381f
 +    @Nullable
 +    @Override
 +    public UUID getId() {
-+        return profile.getId();
++        return profile.getId() != Util.NIL_UUID ? profile.getId() : null;
```

Comment:
I think doing an equals instead of a ref equality for the uuid check would be better to have a consistent state across the server since vanilla and spigot does that as well.